app/ng/views/recipe-details/recipe-details.html

```diff
@@ -0,0 +1,57 @@
+<div class="col-sm-12">
+    <div class="recipedetail-overview">
+        <div class="row">
+            <div class="col-md-12">
+                <img class="headerpicture" ng-src="{{recipe.picture}}"
+                     alt="Titel Foto von Gericht">
+                <span class="headerrecipedetails">{{recipe.title}}</span>
+            </div>
+        </div>
+        <div class="row functionbar">
+            <div class="col-md-3">
+                <button type="button" class="btn btn-primary btn-xs buttonrecipedetail" ui-sref="mealplanner">ADD TO MEAL PLANNER</button>
+            </div>
+            <div class="col-md-3">
```

Comment:
Die column-Struktur entspricht nicht dem Mockups. Dort sind die Vegetarisch/vegan sowie die supermarket-availability info in der selben spalte wie die ingredients darunter. Im momentanen Stand sind die ingredients in einer size-4-spalte, die availability und vegan/vegetarisch-info in einer size-3-spalte.

Lösungsvorschlag:
Unterteilung der buttons und vegan/vegetarisch/availability row auch in eine size-8-column und eine size-4-column. Alle drei buttons dann in die erste column. Dann wäre auch der Abstand zwischen den buttons "natürlicher".